### PR TITLE
[CI]【Hackathon 9th Sprint No.14】功能模块  fastdeploy/rl/rollout_model.py 单测补充

### PR DESCRIPTION
**Pull request by kesmeey**
## Motivation

NO 14  fastdeploy/rl/rollout_model.py 单测补充

image
dedevelop分支 ，覆盖率42，miss 行数 144行(81, 102, 123-131, 146, 155-210, 235, 255->261, 286-287, 294-295, 297-298, 305->307, 331, 340-365, 380, 389-439, 454, 462-487, 502, 510-535, 550, 559-615)

image

当前pr，覆盖率91%，miss 行数 18 

完成覆盖行数  144 - 18 =126

## Modifications

NO 14  fastdeploy/rl/rollout_model.py 单测补充

## Usage or Command

add unittest tests/rl/test_rollout_model.py

No need

## Accuracy Tests

No need

## Checklist

- [x] Add at least a tag in the PR title.
  - Tag list: [`[FDConfig]`,`[APIServer]`,`[Engine]`, `[Scheduler]`, `[PD Disaggregation]`, `[Executor]`, `[Graph Optimization]`, `[Speculative Decoding]`, `[RL]`, `[Models]`, `[Quantization]`, `[Loader]`, `[OP]`, `[KVCache]`, `[DataProcessor]`, `[BugFix]`, `[Docs]`, `[CI]`, `[Optimization]`, `[Feature]`, `[Benchmark]`, `[Others]`, `[XPU]`, `[HPU]`, `[GCU]`, `[DCU]`, `[Iluvatar]`, `[Metax]`]
  - You can add new tags based on the PR content, but the semantics must be clear.
- [x] Format your code, run `pre-commit` before commit.
- [x] Add unit tests. Please write the reason in this PR if no unit tests.
- [x] Provide accuracy results.
- [x] If the current PR is submitting to the `release` branch, make sure the PR has been submitted to the `develop` branch, then cherry-pick it to the `release` branch with the `[Cherry-Pick]` PR tag.

### CODE REVIEW

**Comment by kesmeey** [comment]
image
当前失败的测试并不是当前pr的改动的文件

File: tests/rl/test_rollout_model.py [hunk 1]
```diff
@@ -0,0 +1,211 @@
+import os
+import sys
+import types
+
+import pytest  # type: ignore
+
+# 保守处理：本地缺少 paddle 时直接跳过整模块，CI 上有完整环境会正常运行
```
Comment (CSWYF3634076) [inline tests/rl/test_rollout_model.py]: 可以修改为英文注释，下面同理

File: tests/rl/test_rollout_model.py [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+        parallel_config=parallel_config or types.SimpleNamespace(tensor_parallel_size=1),
+        quant_config=types.SimpleNamespace(name=lambda: quant_name),
+    )
+    inst.state_dict = lambda: {k: 0 for k in state_keys}
+    return inst
+
+
+def test_rollout_model_quantization_and_state_dict_fallback():
+    """RolloutModel wrapper should safely delegate to underlying rollout_model."""
+    # �?get_quantization_infer_keys / state_dict，走默认分支
```
Comment (CSWYF3634076) [inline tests/rl/test_rollout_model.py]: 这里的注释好像是乱码了

File: scripts/coverage_run.sh [hunk 3]
```diff
@@ -14,9 +14,16 @@ failed_tests_file="failed_tests.log"
...
+# 使用 pytest 的 --collect-only 输出，并从每行中提取真正的测试文件路径（形如 tests/.../test_*.py）。
+# 注意：pytest 在收集失败时会输出形如 "ERROR tests/xxx/test_xxx.py::test_xxx ..." 的行，
+# 为了避免把前缀 "ERROR"/"FAILED"/"collecting" 等误当成文件名，这里只保留行中出现的
+# "tests/.../test_*.py" 这一段，其他前后内容直接丢弃。
+TEST_FILES=$(
+  python -m pytest --collect-only -q -c "${PYTEST_INI}" "${tests_path}" --rootdir="${run_path}" --disable-warnings 2>&1 \
+    | grep -E 'tests/.+\/test_.*\.py' \
+    | sed -E 's@.*(tests/[^: ]*test_[^: ]*\.py).*@\1@' \
+    | sort -u
+)
```
Comment (EmmonsCurse) [inline scripts/coverage_run.sh:26]: The new implementation can correctly distinguish test files with the same filename (for example, in different directories[tests/A/test_a.py with tests/B/test_a.py]) and will no longer merge them due to sort/uniq. This is an effective and necessary fix.

**Review by EmmonsCurse — APPROVED** [review]
LGTM ~ The new implementation can correctly distinguish test files with the same filename.

**Comment by CSWYF3634076** [comment]
0.1⭐️
@luotao1
